azkaban-common/src/main/java/azkaban/project/JdbcProjectLoader.java

```diff
@@ -16,6 +16,18 @@
 
 package azkaban.project;
 
+import azkaban.database.AbstractJdbcLoader;
```

Comment:
That's cool. I don't intend to merge this PR. I just wanted to have this patch available in case it becomes useful later on.